plutus-benchmark/validation/BenchCek.hs

```diff
@@ -22,5 +20,6 @@ main = benchWith mkCekBM
    mkCekBM file program =
        -- don't count the undebruijn . unflat cost
        -- `force` to try to ensure that deserialiation is not included in benchmarking time.
-       let !dbterm' = force (unDeBruijnAnonTerm $ unsafeUnflat file program)
-       in whnf (UPLC.unsafeEvaluateCekNoEmit PLC.defaultCekParameters) dbterm'
+       let !nterm = force (toNamedDeBruijnTerm $ unsafeUnflat file program)
+       in whnf unsafeEvaluateCekNoEmit' (throughCheckScope nterm)
```

Comment:
I don't think the scope check should be here? That should be in the full one but not this one, I think?

Reply:
u mean I should either remove throughCheckScope or move throughCheckScope inside nterm/force? I don't think it matters, but I will move it inside `!nterm`

Reply:
Remove it, I think. Otherwise we're not benchmarking the CEK machine purely, but also some other stuff.